### PR TITLE
Api 917 deployment noisiness

### PR DESCRIPTION
**Pull request by jhulbert-va**
https://vajira.max.gov/browse/API-917

Example Jenkins Deployment:
https://tools.health.dev-developer.va.gov/jenkins/view/Shanktopus/job/department-of-veterans-affairs/job/health-apis-deployer/job/API-917-deployment-noisiness/8/console

### CODE REVIEW

**Comment by bschofield-va** [comment]
The output is a little weird
```
[2020-05-20T21:33:14.138Z] b'<DeleteRuleResponse xmlns="http://elasticloadbalancing.amazonaws.com/doc/2015-12-01/">\n  <DeleteRuleResult/>\n  <ResponseMetadata>\n    <RequestId>65abda57-738a-4a83-be8d-3f05a3366c42</RequestId>\n  </ResponseMetadata>\n</DeleteRuleResponse>\n'
```

I'm not really going to complain because it does give us some information that we don't have. But the leading `b` and escaped newlines is a little gross

**Comment by bschofield-va** [comment]
I suspect the next step is correcting/failing the build when something goes wrong

**Comment by jhulbert-va** [comment]
> The output is a little weird
> 
> ```
> [2020-05-20T21:33:14.138Z] b'<DeleteRuleResponse xmlns="http://elasticloadbalancing.amazonaws.com/doc/2015-12-01/">\n  <DeleteRuleResult/>\n  <ResponseMetadata>\n    <RequestId>65abda57-738a-4a83-be8d-3f05a3366c42</RequestId>\n  </ResponseMetadata>\n</DeleteRuleResponse>\n'
> ```
> 
> I'm not really going to complain because it does give us some information that we don't have. But the leading `b` and escaped newlines is a little gross

I was able to use sed to remove these.

https://tools.health.dev-developer.va.gov/jenkins/job/department-of-veterans-affairs/job/health-apis-deployer/job/API-917-deployment-noisiness/10/console

File: bin/load-balancer [hunk 1]
```diff
@@ -246,15 +246,25 @@ createRule() {
...
     echo "Removing old version of rule $RULE_PATH"
     for ruleArn in $ruleArns
     do
-      echo "Deleting rule $ruleArn"
-      aws elbv2 delete-rule --rule-arn "$ruleArn" &
+      echo "Deleting rule $ruleArn"  
+      aws elbv2 delete-rule \
+        --debug \
+        --rule-arn "$ruleArn" \
+        2> >(grep -A 1 "Response body" | tail -1 | sed -e 's/b\(.*\)/\1/' -e 's/\\n//g')
```
Comment (bschofield-va) [inline bin/load-balancer:267]:
```
$ echo "a big boi bug" | sed -e 's/b\(.*\)/\1/'
a ig boi bug
```

Comment (bschofield-va) [reply bin/load-balancer:267]:
```
$ echo -e "a big boi bug\nbetter" | sed -e 's/^b\(.*\)/\1/'
a big boi bug
etter
```

Comment (jhulbert-va) [reply bin/load-balancer]: https://tools.health.dev-developer.va.gov/jenkins/job/department-of-veterans-affairs/job/health-apis-deployer/job/API-917-deployment-noisiness/12/console

**Pull request merged**